.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
This workflow works correctly now. It includes all files in the root directory, and uses the version in the filename of the ZIP, which is catched from the ``version.txt`` file.